core/src/main/java/com/homeaway/streamingplatform/db/dao/impl/StreamDaoImpl.java

```diff
@@ -120,12 +126,18 @@ public void upsertStream(Stream stream) {
             log.error("Stream '{}' is not valid", stream.getName());
         }
 
+        // register schemas
```

Comment:
Add a TODO to support multi-type per stream (for event sourcing and clickstream support)

Reply:
done